tests/providers/test_address.py

```diff
@@ -230,14 +222,6 @@ def test_region(self):
         assert isinstance(region, string_types)
         assert region in GrProvider.regions
 
-    def test_latlng(self):
```

Comment:
I removed this test because there is no top-level latlng provider, and this test acts like there is one! There is one (undocumented) in a locale, but not at this level.